### PR TITLE
rvgl-launcher: Add version 0.1.22.314a1

### PR DESCRIPTION
**Pull request by Zliced13**
Closes https://github.com/ScoopInstaller/Extras/issues/10298

- [x] I have read the [Contributing Guide](https://github.com/Calinou/scoop-games/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by anderlli0053** [comment]
Thanks!

**Comment by Zliced13** [comment]
@Calinou the PowerShell/Windows PowerShell tests are failing for some reason even though my manifest has no trailing white spaces.

I looked at the tests and saw that the **racelab** manifest has a trailing white space in it. Could you fix it please?

**Comment by Calinou** [comment]
> I looked at the tests and saw that the racelab manifest has a trailing white space in it. Could you fix it please?

Should be fixed by https://github.com/Calinou/scoop-games/commit/9d90c9549e62bde4b44490a41b5d294bf5754a09 (if having no `installer` line is equivalent to an empty one… I don't remember).

File: bucket/rvgl-launcher.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+    "version": "0.1.22.314a1",
+    "homepage": "https://www.revoltworld.net/dl/rvgllauncher/",
+    "description": "A cross-platform installer, launcher and package manager for RVGL.",
```
Comment (Calinou) [inline bucket/rvgl-launcher.json:4]:
```suggestion
    "description": "A cross-platform installer, launcher and package manager for RVGL",

```

File: bucket/rvgl-launcher.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{
+    "version": "0.1.22.314a1",
+    "homepage": "https://www.revoltworld.net/dl/rvgllauncher/",
+    "description": "A cross-platform installer, launcher and package manager for RVGL",
+
```
Comment (Calinou) [inline bucket/rvgl-launcher.json:5]:
```suggestion
    "description": "A cross-platform installer, launcher and package manager for RVGL",
```

**Review by Calinou — APPROVED** [review]
Thanks!

**Comment by Zliced13** [comment]
Your welcome.

**Pull request merged**